### PR TITLE
Better italian translation

### PR DESCRIPTION
**Pull request by TheBossMagnus**
**Details of Changes**
Better italian translation for fix bad syntax and for be more similar to windows stock explorer.

Work in progres i'm at line 300

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by THEBOSSMAGNUS** [comment]
Stupid question. i made changes in my forked repo. how i bringh it here?

**Comment by gave92** [comment]
Yes you need to keep ":" where present in the English version.

**Comment by gave92** [comment]
> There are many dupes, can be useful made a separate commints for cleanup?

Examples? But to, you can't simply remove duplicates from the .XLF file, you'd need to also make sure that the removed string are not referenced anywhere in the app.

**Comment by THEBOSSMAGNUS** [comment]
> Examples? But to, you can't simply remove duplicates from the .XLF file, you'd need to also make sure that the removed strings are not referenced anywhere in the app.

ohh it's true. So there are about 5/6 string fore file and some more. I'm turned off pc sorry

**Comment by yaichenbaum** [comment]
@THEBOSSMAGNUS we're working on reducing the number of duplicates.

**Comment by THEBOSSMAGNUS** [comment]
> Examples? But to, you can't simply remove duplicates from the .XLF file, you'd need to also make sure that the removed strings are not referenced anywhere in the app.
can you make an issues were report duplicate string

**Comment by THEBOSSMAGNUS** [comment]
The number of duplicate is very hight: I think more thath 50, probably arround 75-100

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 1]
```diff
@@ -216,15 +216,15 @@
         </trans-unit>
         <trans-unit id="WelcomeDialog.Title" translate="yes" xml:space="preserve">
           <source>Welcome to Files!</source>
-          <target state="translated">Benvenuti in Files!</target>
+          <target state="translated">Benvenuo in Files!</target>
```
Comment (AlexDicy) [inline src/Files/MultilingualResources/Files.it-IT.xlf:219]: Fix the typo "Benvenuo" -> "Benvenuto"

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 2]
```diff
@@ -488,19 +488,19 @@
...
+          <target state="translated">Sei sicuro di voler cancellare permanentemente questi elementi?</target>
         </trans-unit>
         <trans-unit id="ConfirmEmptyBinDialogTitle" translate="yes" xml:space="preserve">
           <source>Empty recycle bin</source>
           <target state="translated">Svuota il Cestino</target>
         </trans-unit>
         <trans-unit id="Yes" translate="yes" xml:space="preserve">
           <source>Yes</source>
-          <target state="translated">OK</target>
+          <target state="translated">sì</target>
```
Comment (AlexDicy) [inline src/Files/MultilingualResources/Files.it-IT.xlf:499]: Shouldn't this be capitalized? "sì" -> "Sì"

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 3]
```diff
@@ -884,11 +884,11 @@
         </trans-unit>
         <trans-unit id="Drives" translate="yes" xml:space="preserve">
           <source>Drives</source>
-          <target state="translated">Risorse</target>
+          <target state="translated">Dischi</target>
         </trans-unit>
         <trans-unit id="EjectNotificationHeader" translate="yes" xml:space="preserve">
           <source>Safe to remove hardware</source>
-          <target state="translated">Il dispositivo può essere rimosso senza problemi</target>
+          <target state="translated">è sicuro rimuovere il dispositivo</target>
```
Comment (AlexDicy) [inline src/Files/MultilingualResources/Files.it-IT.xlf:891]: Capitalize first letter
```suggestion
          <target state="translated">È sicuro rimuovere il dispositivo</target>
```

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 4]
```diff
@@ -1044,7 +1044,7 @@
         </trans-unit>
         <trans-unit id="PropertyEncodingBitrate" translate="yes" xml:space="preserve">
           <source>Encoding Bitrate</source>
-          <target state="translated">Velocità in bit</target>
+          <target state="translated">Bitrate</target>
```
Comment (AlexDicy) [inline src/Files/MultilingualResources/Files.it-IT.xlf:1047]: I suggest not changing this translation, at it is the same as the original
image
```suggestion
          <target state="translated">Velocità in bit</target>
```

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 5]
```diff
@@ -1140,27 +1140,27 @@
         </trans-unit>
         <trans-unit id="PropertyProviderStyle" translate="yes" xml:space="preserve">
           <source>Provider Style</source>
-          <target state="translated">Stile di musica</target>
+          <target state="needs-review-translation">Stile </target>
```
Comment (AlexDicy) [inline src/Files/MultilingualResources/Files.it-IT.xlf:1143]: Remove trailing space
```suggestion
          <target state="needs-review-translation">Stile</target>
```

File: src/Files/MultilingualResources/Files.it-IT.xlf (hunk 1 above)
Comment (THEBOSSMAGNUS) [reply src/Files/MultilingualResources/Files.it-IT.xlf:219]: Thx

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 6]
```diff
@@ -312,19 +312,19 @@
...
         </trans-unit>
         <trans-unit id="SettingsNavExperimental.Content" translate="yes" xml:space="preserve">
           <source>Experimental</source>
-          <target state="translated">Esperimenti</target>
+          <target state="translated">Funzioni sperimentali</target>
         </trans-unit>
         <trans-unit id="ItemAlreadyExistsDialogContent" translate="yes" xml:space="preserve">
           <source>An item with this name already exists in this directory.</source>
-          <target state="translated">Un elemento con questo nome esiste già in questa cartella.</target>
+          <target state="translated">Un elemento lo stesso nome esiste già in questa cartella.</target>
```
Comment (gave92) [inline src/Files/MultilingualResources/Files.it-IT.xlf:327]: `Un elemento *con* lo stesso nome`

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 7]
```diff
@@ -1112,19 +1112,19 @@
         </trans-unit>
         <trans-unit id="PropertyContentDistributor" translate="yes" xml:space="preserve">
           <source>Content Distributor</source>
-          <target state="translated">Provider di contenuti</target>
+          <target state="translated">Ditributore del contenuto</target>
```
Comment (gave92) [inline src/Files/MultilingualResources/Files.it-IT.xlf:1115]: Typo: *Distributore*

Comment (THEBOSSMAGNUS) [reply src/Files/MultilingualResources/Files.it-IT.xlf]: Done thanks

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 8]
```diff
@@ -216,15 +216,15 @@
...
+          <target state="translated">Benvenuto in Files!</target>
         </trans-unit>
         <trans-unit id="WelcomeDialog.PrimaryButtonText" translate="yes" xml:space="preserve">
           <source>Grant Permission</source>
           <target state="translated">Concedi i permessi</target>
         </trans-unit>
         <trans-unit id="WelcomeDialogTextBlock.Text" translate="yes" xml:space="preserve">
           <source>To get started, you'll need to grant us permission to display your files. This will open a Settings page where you can grant us this permission. You'll need to reopen the app once you've completed this step. </source>
-          <target state="translated">Per iniziare, devi concederci l'autorizzazione per visualizzare i tuoi file. Si aprirà una pagina Impostazioni in cui puoi concederci questa autorizzazione. Dovrai riaprire l'app ad operazione completata.</target>
+          <target state="translated">Per iniziare, devi concederci l'autorizzazione per visualizzare i tuoi file. Si aprirà una pagina Impostazioni in cui puoi concederci questa autorizzazione.Sarà necessario riaprire l'app quando avrai concesso l'autorizzazione.</target>
```
Comment (gave92) [inline src/Files/MultilingualResources/Files.it-IT.xlf:227]: Manca uno spazio: `autorizzazione. Sarà necessario`

**Comment by THEBOSSMAGNUS** [comment]
I've finished all of 876 strings, probably some errors still exist, but I think that everything is more understandable and similar to stock-explorer.

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 9]
```diff
@@ -3238,7 +3238,7 @@ Usiamo App Center per tenere traccia dell'utilizzo dell'app, trovare bug e risol
         </trans-unit>
         <trans-unit id="RestoreDefault" translate="yes" xml:space="preserve">
           <source>Restore default</source>
-          <target state="translated">Ripristina prederfinita</target>
+          <target state="translated">Ripristina prederfiniti</target>
```
Comment (gave92) [inline src/Files/MultilingualResources/Files.it-IT.xlf:3241]: Typo: `predefiniti`

File: src/Files/MultilingualResources/Files.it-IT.xlf [hunk 10]
```diff
@@ -3250,7 +3250,7 @@ Usiamo App Center per tenere traccia dell'utilizzo dell'app, trovare bug e risol
         </trans-unit>
         <trans-unit id="HighDpiOption_UseDPIOnLogin" translate="yes" xml:space="preserve">
           <source>On Windows login</source>
-          <target state="translated">Al login di Windows</target>
+          <target state="translated">Al avvio di Windows</target>
```
Comment (gave92) [inline src/Files/MultilingualResources/Files.it-IT.xlf:3253]: `All'avvio`

**Review by gave92 — CHANGES_REQUESTED** [review]

File: src/Files/MultilingualResources/Files.it-IT.xlf (hunk 8 above)
Comment (THEBOSSMAGNUS) [reply src/Files/MultilingualResources/Files.it-IT.xlf]: done, thank you!

**Review by gave92 — APPROVED** [review]
Thanks!

**Comment by TheBossMagnus** [comment]
@gave92 @yair100 

There is someting else that i can be useful in?
I don't know c#/c++, only powershell and batch.

**Pull request merged**